docs/applications/edit-application.md

```diff
@@ -17,8 +17,7 @@ required_permissions:
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
 1. Click `DETAILS` on the application you want to edit
-1. On the upper right corner of the application details view click `...`
-1. Use the drop-down menu to choose `EDIT`
+1. On the upper right corner of the application details view click `EDIT`
```

Comment:
@guipontes7 could you apply changes below to help us with consistent use of commas on this type of instruction?

```suggestion
1. On the upper right corner of the application details view, click `EDIT`
```

Reply:
File updated!